ci/install-build-deps.sh

```diff
@@ -128,19 +128,19 @@ else
     curl -Lo ~/.cargo/bin/rustup 'https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-init'
     chmod +x ~/.cargo/bin/rustup
     hash -r
+    rustup self update # Create the hardlinks for cargo, rustc, etc
```

Comment:
You could download `rustup` if it does not exist and then `rustup self update`, instead of checking if it does and repeating the `rustup self update`.

Reply:
Done.